src/components/status/status.scss

```
@@ -61,6 +63,10 @@
 
 .sbb-status__title {
   margin-block: 0;
+
+  :host(:not([data-slot-names~='title'], [title-content])) & {
```

would it not be more consistent?
```suggestion
  :host(:not(#{$has-title}) & {
```

---

Maybe yes :/
But, to be consistent, we must change all of the occurrences

---

in the status.scss file I only see one occurrence. the others in this file do already use $has-title.

---

ah... my bad :/
i thought you wanted to change every file